env/HERA.env

```diff
@@ -107,6 +107,12 @@ elif [[ "${step}" = "ocnanalchkpt" ]]; then
 
     export APRUN_OCNANAL="${launcher} -n ${npe_ocnanalchkpt}"
 
+elif [[ "${step}" = "ocnanalecen" ]]; then
+
+    export APRUNCFP="${launcher} -n \$ncmd --multi-prog"
+
+    export APRUN_OCNANALECEN="${launcher} -n ${npe_ocnanalecen}"
```

Comment:
I think we need to add the `--cpus-per-task` argument to prevent oversubscribing

Reply:
added, hopefully in a helpful fashion